tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -89,10 +104,6 @@ def test_property_missing(nhsd_apim_proxy_url, property, pointer, correlation_id
     )
 
 
-"""
-Null data 400 test
-"""
-
 _null_properties = [
```

Comment:
Should be using null props from constants, look at the other env validation tests